content/guides/getting-started/component-framework-configuration.md

```diff
@@ -29,13 +29,13 @@ Testing, the app will automatically guide you and set up your configuration.
 Cypress offers **automatic configuration** for supported front-end frameworks
 and bundlers.
 
-For first-time setup of React of Vue apps, we strongly recommend using automatic
+For first-time setup of React or Vue apps, we strongly recommend using automatic
 configuration to create all of the necessary configuration files.
 
 ### Automatic Configuration Setup
 
 You should use the Cypress Launchpad to configure Component Testing for the
-first time. To follow the UI prompts displayed the Launchpad, simply open
+first time. To start the setup wizard, simply open
 Cypress and choose "Component Testing".
```

Comment:
Would it be worth mentioning that they can launch with this as well:

```shell
cypress open --component
```

Reply:
I think in this case they both do the same thing as it launches automatic config. We get more into cli flag in more advanced guides.